streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -208,6 +208,22 @@ public synchronized void close() {
     }
 
     /**
+     * Produces a string representation contain useful information about Kafka Streams
+     * Such as thread IDs, task IDs and a representation of the topology. This is useful
+     * in debugging scenarios.
+     * @return A string representation of the Kafka Streams instance.
+     */
+    public String toString() {
+        StringBuilder sb = new StringBuilder("KafkaStreams processID:" + this.processId + "\n");
+        for (int i = 0; i < this.threads.length; i++) {
+            sb.append(this.threads[i].toString());
```

Comment:
I've done the indentation internally, prints fine for me. Same applies to the other indentation comments.